samples/net/sockets/echo_client/sample.yaml

```diff
@@ -11,6 +11,9 @@ tests:
   sample.net.sockets.echo_client:
     platform_allow: qemu_x86 frdm_k64f sam_e70_xplained
       qemu_cortex_m3 frdm_kw41z
+  sample.net.sockets.echo_client.6locan:
+    extra_args: OVERLAY_CONFIG="overlay-6locan.conf"
+    platform_allow: nucleo_f746zg
```

Comment:
is this the only platform supporting this?

Reply:
Actually all with can and sufficient ram/rom should work.